packages/notifications/src/elements/content/Title.spec.tsx

```diff
@@ -16,4 +20,88 @@ describe('Title', () => {
 
     expect(container.firstChild).toBe(ref.current);
   });
+
+  describe('Types', () => {
```

Comment:
Awesome, thanks 🙏. I missed testing this in the CSS-in-JS migration for `react-notifications`.